lib/cli/src/commands/app/volumes/credentials/rotate_secrets.rs

```diff
@@ -0,0 +1,36 @@
+use crate::{
+    commands::{app::util::AppIdentOpts, AsyncCliCommand},
+    config::WasmerEnv,
+};
+
+use colored::Colorize;
+
+/// Rotate the secrets linked to volumes of an app.
+#[derive(clap::Parser, Debug)]
+pub struct CmdAppVolumesRotateSecrets {
+    #[clap(flatten)]
+    pub env: WasmerEnv,
+
+    #[clap(flatten)]
+    pub ident: AppIdentOpts,
+}
+
+#[async_trait::async_trait]
+impl AsyncCliCommand for CmdAppVolumesRotateSecrets {
+    type Output = ();
+
+    async fn run_async(self) -> Result<Self::Output, anyhow::Error> {
+        let client = self.env.client()?;
+        let (_ident, app) = self.ident.load_app(&client).await?;
+
+        wasmer_api::query::rotate_s3_secrets(&client, app.id).await?;
+
+        println!(
+            "Correctly rotated s3 secrets for app {} ({})",
```

Comment:
When you rotate you'll pretty much always want to see the new secrets, no?

Would make sense for the command to also show the new values.

Reply:
([5f80b94](https://github.com/wasmerio/wasmer/pull/5070/commits/5f80b948eba4ed1f9989676560388ee280135fc2))